### PR TITLE
"Scoring system with player before merge" 

### PR DESCRIPTION
**Pull request by Mushroom-MSL1L**
1. request merge "Scoring system with player before merge" branch into "dev"
2. this branch has already merge "dev" into branch of "scoring_system" (hope these's no conflict)
3. have functions about player
4. change url to "api" prefix
5. merge files in "translate" path into files in "endpoint" path
** hope Jason at least review the part about router, participant, user**

### CODE REVIEW

**Review by JasonTuTW — APPROVED** [review]

File: go.work [hunk 1]
```diff
@@ -0,0 +1,3 @@
+go 1.20
+
+use ./backend
```
Comment (lolainta) [inline go.work]: We don't need `go.work` and `go.work.sum`

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (lolainta) [inline .gitignore]: Not needed

**Review by lolainta — CHANGES_REQUESTED** [review]
- Do not commit unrelated files, such as archery_game_scoring folder.
- Make sure that all files are up to date with dev branch. (README.md are modified)

File: .gitignore
Comment (lolainta) [inline .gitignore:1]: This file is needed

**Review by lolainta — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (Mushroom-MSL1L) [reply .gitignore]: Did I misunderstand?
You just want to delete a blank line, right?

**Review by lolainta — APPROVED** [review]

**Pull request merged**